dashboard/app/graphs.go

```diff
@@ -199,7 +207,13 @@ func handleCoverageHeatmap(c context.Context, w http.ResponseWriter, r *http.Req
 	dateTo := civil.DateOf(time.Now())
 	var style template.CSS
 	var body, js template.HTML
-	if style, body, js, err = cover.DoHeatMapStyleBodyJS("syzkaller", hdr.Namespace, dateFrom, dateTo); err != nil {
+	f := cover.DoHeatMapStyleBodyJS
+	switch heatmapType {
+	case "dir":
```

Comment:
Nit: assign `DoHeatMapStyleBodyJs` in `default:` or do it under `case "dir":` and return an error in `default:`?

Reply:
var f .... requires the full function declaration and it was quite long.
If it looks too strange - I'll add the full declaration.